src/server/src/main/java/io/cassandrareaper/core/RepairSegment.java

```diff
@@ -130,9 +130,11 @@ public Builder reset() {
   }
 
   public enum State {
+    // don't change order or insert items (ordial is used in db)
     NOT_STARTED,
     RUNNING,
-    DONE
+    DONE,
+    STARTED
```

Comment:
UI needs updating to for this.

Reply:
I pushed a commit to address this.

Reply:
thanks @adejanovski 

still working on fixing the upgrade integration test. just haven't had as much time as i'd hope for this week :(

Reply:
was addresses in #628 (the upgrade integration tests have been disabled for now)